### ISSUE TITLE
Encrypt-all is not working

### ISSUE THREAD

**Issue by Heracles31** [nextcloud/docker#576]
Hi,

For my first deployment, I used Nextcloud as a snap. I now switched to Docker. Both format have the same problem : the occ command is blocked in a catch22. I posted the snap issue on that page but never received any reply.

https://help.nextcloud.com/t/nextcloud-snap-users-please-list-the-issues-youre-facing/1979/144

After turning On encryption, the command to encrypt everything is "occ encryption:encrypt-all" and itself is supposed to be executed only once in maintenance mode to prevent users from changing files while the encryption is in progress.

The thing is, the encrypt-all command does not work BECAUSE of the maintenance mode. It complains that because of maintenance mode, no module are loaded, so encryption module is not loaded.

The command does encrypt all as expected if out of maintenance mode.

The encrypt-all command need to be usable while in maintenance mode.

**Comment by J0WI** [comment]
This is not related to docker or snap. Closing as duplicate of https://github.com/nextcloud/server/issues/8932
